manifests/gcp_marketplace/schema.yaml

```diff
@@ -100,20 +100,23 @@ properties:
     type: string
     x-google-marketplace:
       type: NAMESPACE
-  serviceAccountCredential:
-    title: |-
-     GCP Service Account credentials used to call other GCP services.
-    description: |-
-      This deployment requires a service account to use for authentication when calling other GCP services. 
-      Specify the base64-encoded credentials for the service account you want to use. 
-      You can get these credentials by running the following command in a terminal window. 
-      \"$ gcloud iam service-accounts keys create application_default_credentials.json --iam-account [your-service-account] && cat application_default_credentials.json | base64\"
-      If you are using Linux, please use \"base64 -w 0\" to disable line wrapping.
-    type: string
-    default: ""
-    x-google-marketplace:
-      type: STRING
+# TODO: temporarily remove sa credential from deploy page in MKP. SA will be added by user manually
```

Comment:
I think we can permanently remove SA credential box, right? The long term plan is to have our own configuration page. I don't see us bringing back SA credential box.

Reply:
Got it. Done.